pages/linux/xmount.md

```diff
@@ -0,0 +1,17 @@
+# xmount
+
+> Convert on-the-fly between multiple input and output hard disk image types with optional write cache support.
+> Creates a virtual file system using FUSE (Filesystem in Userspace) that contains a virtual representation of the input image.
+> More information: <https://www.pinguin.lu/xmount>.
```

Comment:
```suggestion
> More information: <https://manned.org/xmount>.
```
Maybe we should use this link, since the official page doesn't really provide any useful information